src/pages/finalreview.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <q-page padding>
+    <div class="q-pa-md">
+    <q-markup-table flat bordered>
+      <thead class="bg-teal">
+        <tr>
+          <th colspan="6">
+            <div class="row no-wrap items-center">
+              <q-img
+                style="width: 70px"
+                :ratio="1"
+                class="rounded-borders"
+              />
+
+              <div class="text-h4 q-ml-md text-white">Ladang Saya</div>
+            </div>
+          </th>
+        </tr>
+        <tr>
+          <th class="text-left">Ternakan/Tanaman</th>
+          <th class="text-right">Berat(KG)</th>
+          <th class="text-right">Jantina(M/F)</th>
+          <th class="text-right">Keluasan Diusahakan Megikut Jenis Tanaman(Hektar)</th>
+          <th class="text-right">Hasil Di Ladang($/KG)</th>
+          <th class="text-right">Tarikh</th>
+        </tr>
+      </thead>
+      <tbody class="bg-grey-3">
+        
+         <tr>
+          <td class="text-left">Kerbau</td>
+          <td class="text-right">10</td>
+          <td class="text-right">M</td>
+          <td class="text-right">-</td>
+          <td class="text-right">5.5</td>
+          <td class="text-right">11/9/2021</td>
+        </tr>
```

Comment:
Just taking the first 2 rows as reference. We can either add another column to include "Edit" ✏ and "Delete" 🚮 actions on each row. Or do you think users would understand to double-click a row and pop-up edit dialog will allow them to edit on the spot? E.g. https://quasar.dev/vue-components/table#popup-editing

You decide.

Reply:
this will take me a while to implement

Reply:
Since you decide to do popup, you should still at least add a delete button to each row.